sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -382,9 +413,36 @@ static RunQueryRequest makeRequest(Query query, @Nullable String namespace) {
     }
 
     /**
+     * Returns a new {@link DatastoreV1.Read} that reads the results of the specified GQL query.
+     * See <a href="https://cloud.google.com/datastore/docs/reference/gql_reference">GQL Reference
+     * </a> to know more about GQL grammar.
+     */
+    @Experimental(Kind.SOURCE_SINK)
```

Comment:
Why do you need to mark the method experimental - isn't the whole class already experimental? (I'm not sure why the class should be, either... I think we're past the point where IO is experimental)